### ISSUE TITLE
[PROPOSAL] Standardize code style for UI using prettier

### ISSUE THREAD

**Issue by harupy** [mlflow/mlflow#2560]
### Describe the proposal

Introduce [Prettier](https://prettier.io/) to standardize code style for the UI.

### Motivation

Prettier allows everyone to format code in the same way and helps improve development productivity. We can avoid wasted efforts during code review.

### Proposed Changes

I'm planning to follow the steps proposed in https://github.com/kubeflow/pipelines/issues/2406.

- [ ] Make a reasonable prettier config that's consistent with existing code style.
- [ ] Transform all code with prettier after syncing with people having open PRs.
- [ ] Set up CI check.
- [ ] Update CONTRIBUTING doc with related information.

**Comment by smurching** [comment]
Thanks @harupy! Some additional context from offline discussion: `prettier` (when applied via the `prettier-eslint` package) is capable of applying the same rules as `eslint`, & basically can fix a superset of the linter errors that can be fixed via `eslint --fix`, as per discussion in https://github.com/prettier/prettier-eslint/issues/101

**Comment by harupy** [comment]
@smurching Thanks! It seems [prettier/eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier) can do the same thing.

**Comment by harupy** [comment]
I tried the instruction below which seems to work.

https://prettier.io/docs/en/integrating-with-linters.html#eslint

**Comment by harupy** [comment]
Should we foramt css files (`prettier-eslint` and `prettier-plugin-eslint` don't format css)? 

If we do, we can use the commands below.
```
# format
prettier 'src/**/*.css'

# check
prettier --check 'src/**/*.css'
```

**Comment by harupy** [comment]
Resolved by #2561
